examples/src/main/java/org/apache/spark/examples/streaming/JavaDirectKafkaWordCount.java

```diff
@@ -50,25 +52,30 @@
   private static final Pattern SPACE = Pattern.compile(" ");
 
   public static void main(String[] args) throws Exception {
```

Comment:
We should also update the document in the code comment at the beginning of `JavaDirectKafkaWordCount`.

Reply:
viirya, thank you for your nice review.